src/interactive/commands.sml

```diff
@@ -66,14 +66,15 @@ struct
             INL s => raise Fail s
           | INR sign =>
             let
-              val elab = RefineElab.transport o ValidationElab.transport o BindSignatureElab.transport
-              val sign' = elab sign
+              val elab = ValidationElab.transport o BindSignatureElab.transport
+              val sign' = RefineElab.transport (AbtSignature.Telescope.append (oldSign, elab sign))
```

Comment:
Also, we pass `oldSign` to `RefineElab`, but not to `BingSignatureElab`. `BindSignatureElab` takes StringTelescope and returns SymbolTelescope. How to transform SymbolTelescope to StringTelescope to pass it to `BindSignatureElab`?